daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Transaction.scala

```diff
@@ -174,9 +181,18 @@ final case class GenTransaction[Nid, +Cid](
           val node1 = nodes(nid1)
           val node2 = other.nodes(nid2)
           node1 match {
-            case _: Node.NodeRollback[_] =>
-              // TODO https://github.com/digital-asset/daml/issues/8020
-              sys.error("rollback nodes are not supported")
+            case nr1: Node.NodeRollback[Nid] => //TODO: can this be NodeRollback[_] ?
```

Comment:
Why shouldn’t it be able to be a rollback?

Reply:
Sorry, my comment was misleading.... It was a question about scala semantics.
Specifically, is there any semantic difference if we use an underscore instead of `Nid` here?
The sibling matches here use `Nid`, but other places in the code use `_`. Does it matter?